graylog2-server/src/main/java/org/graylog2/alarmcallbacks/EmailAlarmCallback.java

```diff
@@ -172,11 +172,13 @@ public void initialize(Configuration config) throws AlarmCallbackConfigurationEx
     // I am truly sorry about this, but leaking the user list is not okay...
     private ConfigurationRequest getConfigurationRequest(Map<String, String> userNames) {
         ConfigurationRequest configurationRequest = new ConfigurationRequest();
-        configurationRequest.addField(new TextField("sender",
-                "Sender",
-                "",
-                "The sender of sent out mail alerts",
-                ConfigurationField.Optional.OPTIONAL));
+        if (!graylogConfig.isCloud()) {
```

Comment:
This is actually dead code. But it doesn't hurt to do that here as well

Reply:
Indeed, it seems it will go away in 4.1. Until then someone could still create those through the UI I guess, so better safe than sorry :)